src/main/java/org/arl/fjage/remote/ConnectionHandler.java

```diff
@@ -154,7 +153,6 @@ public void run() {
         log.warning("Bad JSON request: "+ex.toString() + " in " + s);
       }
     }
-    fw.authenticate(conn, null);
```

Comment:
Do we need a "logout" call which allows a Firewall to cleanup things when the connection is no longer used?

Reply:
Yeah, I was considering that and sort of forgot about it. Yeah, it's probably good to have some way to clean this up.

I guess we could put that back in, only problem I have with it is that it is currently indistinguishable from `SlaveContainer.authenticate(null)`. Maybe I'm overthinking it? 

Perhaps something like `Firewall.discard()`. Overkill?

Either way is fine, just presenting some options.

Reply:
`Firewall.discard()` (or my preference for `Firewall.signoff()` or something similar) works well.

Reply:
Implemented in 02fd3a7d133b306b9f28a3f06176946d5ef26336